src/library/scala/collection/immutable/Vector.scala

```diff
@@ -466,24 +466,24 @@ override def companion: GenericCompanion[Vector] = Vector
       display5 = copyRange(display5, oldLeft, newLeft)
   }
 
-  private def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
+  private[this] def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
```

Comment:
Actually, I reran the original tests and I'm not 100% sure it makes any difference at all.  If it's important I should rerun them later, maybe with JMH.  I've been trying to iterate quickly, so have been using Thyme which is more subject to fluctuations.